### PR TITLE
Add base-url to sitemap.xml

### LINKED ISSUE #8072: Add base-url to sitemap.xml
Our sitemap is of the following form:
```
<url>
    <loc>/howto/deploy/</loc>
</url>
<url>
    <loc>/quickstart/</loc>
</url>
...
```

But as requested from our SEO expert, it should be changed to the following form, to improve searches:
```
<url>
    <loc>https://docs.lakefs.io/howto/deploy/</loc>
</url>
<url>
    <loc>https://docs.lakefs.io/quickstart/</loc>
</url>
...
```

### PR DESCRIPTION
**Pull request by itaigilo**
Closes #8072.

## Change Description

Adds a base-url to the sitemap.xml.

### New Feature

Our sitemap is of the following form:
```
<url>
    <loc>/howto/deploy/</loc>
</url>
<url>
    <loc>/quickstart/</loc>
</url>
...
```

But as requested from our SEO expert, this PR changes it to the following form, to improve searches:
```
<url>
    <loc>https://docs.lakefs.io/howto/deploy/</loc>
</url>
<url>
    <loc>https://docs.lakefs.io/quickstart/</loc>
</url>
...
```

### Testing Details

I've tested the `sed` command locally - 
But I need to validate this works fine after ~~the next release~~ it's merged to `master`.

### CODE REVIEW

File: .github/workflows/docs-latest.yaml
Comment (itaiad200) [inline .github/workflows/docs-latest.yaml:37]: Will this run for all versions or just the latest? What happens when the latest becomes the not-so-latest? Won't it continue to point to the latest link?

Comment (itaigilo) [reply .github/workflows/docs-latest.yaml:37]: We have two actions: `docs-latest.yaml` and `docs-release.yaml`.

The first is dispatched upon push-to-master, and builds and pushes the default docs (meaning, whatever is in `docs/_site`).
The second is dispatched upon release ("publish"), and builds and pushes the docs for the specific released version, into the version's folder (meaning, into `docs/_site/v1.31/`).

So this new step will run for every merge to master, and will override the existing sitemap.

(But that's a good point - changing the PR description according to this.)

**Comment by itaigilo** [comment]
Note:
The `Docs Preview` check fails for a while now.
We discussed a while ago if it's worth fixing and figured it's not - 
But maybe we should reconsider it, or remove it?

**Review by itaiad200 — APPROVED** [review]
LGTM - remember to inspect it after this merge

**Pull request merged**
